pkg/client/liquidation_recovery_test.go

```diff
@@ -0,0 +1,506 @@
+								"actual:   [%s]",
+							i,
+							bitcoinHandle.transactions[0],
+							transaction,
+						)
+					}
+
+				}
+			case err := <-errChan:
+				t.Fatalf("unexpected error: %v", err)
+			case <-ctx.Done():
+				t.Fatal(ctx.Err())
+			}
+		})
+	}
+}
+
+func generateMemberKeys() ([]tss.MemberID, []common.Address, error) {
+	memberIDs := []tss.MemberID{}
+	memberAddresses := []common.Address{}
+
+	for _, memberIDString := range []string{
+		"04754b25e1b91dc4006acf17d2c28788be8398a8ed591ba2cbbff5c424d23d91971a8881edd3fc64772d90a181665b4b2ffdbbf05776b8fa8bd08893c26c1cad44",
+		"045300560c6c1619d8e2fd4bacc5566c330a89b6402c8c8ceb748d4232b5157dce812ab86645fc66e534a7a3238299eb258245e48a3885d3eea7b885e6c94ddfed",
+		"047279cff18c9bdfad9f6f23407070b9ace75acb5570d687de3416a306ecae16a7b40e6f1721f30bcee9b910e8a3d9bb298e9a6540826cf3ae5fbe1163a60d86ec",
+	} {
+		memberID, err := tss.MemberIDFromString(memberIDString)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		memberIDs = append(memberIDs, memberID)
+		memberAddresses = append(memberAddresses, common.HexToAddress(memberID.String()))
+	}
+
+	return memberIDs, memberAddresses, nil
+}
+
+func initializeSigners() ([]tss.MemberID, []common.Address, map[string]*tss.ThresholdSigner, error) {
+	if err := log.SetLogLevel("*", "INFO"); err != nil {
+		return nil, nil, nil, err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	if err := log.SetLogLevel("*", "DEBUG"); err != nil {
+		return nil, nil, nil, err
+	}
+
+	groupMemberIDs, keepMembersAddresses, err := generateMemberKeys()
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	keep = localChain.OpenKeep(keepAddress, depositAddress, keepMembersAddresses)
+
+	doneChan := make(chan interface{})
+	errChan := make(chan error)
+
+	testData, err := testdata.LoadKeygenTestFixtures(groupSize)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to load key gen test fixtures: [%v]", err)
+	}
+
+	pubKeyToAddressFn := func(publicKey cecdsa.PublicKey) []byte {
+		return elliptic.Marshal(publicKey.Curve, publicKey.X, publicKey.Y)
+	}
+
+	var testWait sync.WaitGroup
+	testWait.Add(groupSize)
+
+	var providersInitializedWg sync.WaitGroup
+	providersInitializedWg.Add(groupSize)
+
+	signers := make(map[string]*tss.ThresholdSigner)
+
+	go func() {
+		for i, memberID := range groupMemberIDs {
+			go func(memberID tss.MemberID, index int) {
+				operatorPublicKey, err := memberID.PublicKey()
+				if err != nil {
+					errChan <- err
+					return
+				}
+				networkPublicKey := key.NetworkPublic(*operatorPublicKey)
+				networkProvider := localNet.ConnectWithKey(&networkPublicKey)
+
+				networkProviders[memberID.String()] = networkProvider
+
+				providersInitializedWg.Done()
+				providersInitializedWg.Wait()
+
+				// FIXME: Load signers from local test data storage instead of
+				// generating them with tss.GenerateThresholdSigner.
+				//
+				// signer, err := newTestSigner(groupMemberIDs, index)
```

Comment:
I think that if we don't have a working implementation at the moment, I'd rather not commit an unused function or commented out function call. I think the FIXME is good though